library/std/src/net/ip.rs

```diff
@@ -1000,8 +995,8 @@ impl From<u32> for Ipv4Addr {
     /// ```
     /// use std::net::Ipv4Addr;
     ///
-    /// let addr = Ipv4Addr::from(0x0d0c0b0au32);
-    /// assert_eq!(Ipv4Addr::new(13, 12, 11, 10), addr);
+    /// let addr = Ipv4Addr::from(0xcafebabe);
+    /// assert_eq!(Ipv4Addr::new(0xca, 0xfe, 0xba, 0xbe), addr);
```

Comment:
Why bother changing these constants?

Reply:
I find it a little easier to read. Probably should be a separate commit.

Reply:
Split to a separate PR: #75175